static/bidder-info/adprime.yaml

```diff
@@ -12,3 +12,10 @@ capabilities:
       - banner
       - video 
       - native
+userSync:
+  redirect:
+    url: "https://sync.adprime.com/pbserver?gdpr={{.GDPR}}&gdpr_consent={{.GDPRConsent}}&ccpa={{.USPrivacy}}&gpp={{.GPP}}&gpp_sid={{.GPPSID}}&redir={{.RedirectURL}}"
+    userMacro: "[UID]"
+  iframe:
+    url: "https://sync.adprime.com/pbserverIframe?gdpr={{.GDPR}}&gdpr_consent={{.GDPRConsent}}&ccpa={{.USPrivacy}}&gpp={{.GPP}}&gpp_sid={{.GPPSID}}&pbserverUrl={{.RedirectURL}}"
+    userMacro: "[UID]"
```

Comment:
@Adprime  iframe didn't append uid at the end for following url:

```
https://sync.adprime.com/pbserverIframe?gdpr=&gdpr_consent=&ccpa=&gpp=&gpp_sid=&pbserverUrl=https%3A%2F%2Fprebid.adnxs.com%2Fpbs%2Fv1%2Fsetuid%3Fbidder%3Dadprime%26gdpr%3D%26gdpr_consent%3D%26f%3Di%26uid%3D%22%5BUID%5D%22
```

![image](https://github.com/prebid/prebid-server/assets/24757781/f5fb1653-5284-4266-bebe-98562a81aeb3)

looks like uuid is present in js script but not correctly appended to redirect url

Reply:
@onkarvhanumante 
is your link valid?
```
https://prebid.adnxs.com/pbs/v1/setuid?bidder=adprime&gdpr=&gdpr_consent=&f=i&uid="[UID]"
```

I corrected it a little
```
https://prebid.adnxs.com/pbs/v1/setuid?bidder=adprime&gdpr=&gdpr_consent=&f=i&uid=[UID]
```

in this case, when calling
```
https://sync.adprime.com/pbserverIframe?gdpr=&gdpr_consent=&ccpa=&gpp=&gpp_sid=&pbserverUrl=https%3A%2F%2Fprebid.adnxs.com%2Fpbs%2Fv1%2Fsetuid%3Fbidder%3Dadprime%26gdpr%3D%26gdpr_consent%3D%26f%3Di%26uid%3D%5BUID%5D
```

everything works correctly
![image](https://github.com/prebid/prebid-server/assets/64427228/46c26443-0c81-437f-a89b-0f9afa138a22)

Reply:
> @onkarvhanumante is your link valid?
> 
> ```
> https://prebid.adnxs.com/pbs/v1/setuid?bidder=adprime&gdpr=&gdpr_consent=&f=i&uid="[UID]"
> ```
> 
> I corrected it a little
> 
> ```
> https://prebid.adnxs.com/pbs/v1/setuid?bidder=adprime&gdpr=&gdpr_consent=&f=i&uid=[UID]
> ```
> 
> in this case, when calling
> 
> ```
> https://sync.adprime.com/pbserverIframe?gdpr=&gdpr_consent=&ccpa=&gpp=&gpp_sid=&pbserverUrl=https%3A%2F%2Fprebid.adnxs.com%2Fpbs%2Fv1%2Fsetuid%3Fbidder%3Dadprime%26gdpr%3D%26gdpr_consent%3D%26f%3Di%26uid%3D%5BUID%5D
> ```
> 
> everything works correctly ![image](https://private-user-images.githubusercontent.com/64427228/344637182-46c26443-0c81-437f-a89b-0f9afa138a22.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTk4Mzc5MjEsIm5iZiI6MTcxOTgzNzYyMSwicGF0aCI6Ii82NDQyNzIyOC8zNDQ2MzcxODItNDZjMjY0NDMtMGM4MS00MzdmLWE4OWItMGY5YWZhMTM4YTIyLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA3MDElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNzAxVDEyNDAyMVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWNjMzVjOGY2MmQyZTY2NGFjZmYxODg5ODc0M2E0OGQzNGI1NDI2ZDZmYWE2ZjBkM2U2ZjlmYjlkYzVkY2QxMTImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.hTSdB5DmoRgc3pD3Q1yyc1IOldP4oU_hZ6tZWzTYs3I)



@Adprime  `"[UID]"` was changed to `[UID]` in your case. But `static/bidder-info/adprime.yaml` has `"[UID]"` as userMacro

Reply:
@onkarvhanumante as far as I understand, we must pass a **string** to `userMacro`. The .yaml specification allows you to specify a string with or without quotes (`"`). If you remove the `"`, the `UID` string will be enclosed in `[` and `]`, which is a **list**. Please correct me if I'm wrong

Reply:
@onkarvhanumante any updates?